src/Native/Unix/System.Security.Cryptography.Native/pal_ssl.cpp

```diff
@@ -14,8 +14,11 @@ static_assert(PAL_SSL_ERROR_WANT_WRITE == SSL_ERROR_WANT_WRITE, "");
 static_assert(PAL_SSL_ERROR_SYSCALL == SSL_ERROR_SYSCALL, "");
 static_assert(PAL_SSL_ERROR_ZERO_RETURN == SSL_ERROR_ZERO_RETURN, "");
 
+extern "C" int32_t CryptoNative_EnsureOpenSslInitialized();
+
 extern "C" void CryptoNative_EnsureLibSslInitialized()
 {
+    CryptoNative_EnsureOpenSslInitialized();
```

Comment:
Can you add a comment that this must come first to ensure the initialization is done in a thread-safe manner?